src/cr/cube/cubepart.py

```diff
@@ -1546,7 +1544,7 @@ class _Nub(CubePartition):
 
     @lazyproperty
     def is_empty(self):
-        return False if self.unweighted_count else True
+        return math.isnan(self.unweighted_count)
```

Comment:
This property needs a docstring that states its contract.

Assuming that `.is_empty` means: "contains no values", I wonder what its semantics are for a non-container object (as I understand a `Nub` to be). Also it's not clear to me the circumstances under which a `Nub` value for unweighted count can be NaN. These are questions I'd like the docstring to answer for me.

Reply:
The integration test that breaks is `integration/test_cube_part.py::Describe_Nub::it_is_empty` which loads https://github.com/Crunch-io/crunch-cube/blob/f9e0b2a4cf81e955e592b4584d532372c1089cce/tests/fixtures/econ-nodata-no-dims.json#L84-L86, which does indeed have `"counts":{null}`. My best guess is it's what happens when the dataset has no rows (or all rows have been filtered/excluded)?

Reply:
Hm, in my testing, a query from a dataset with 0 rows, a dataset excluded down to 0 rows, and a query filtered down to 0 rows all have a 0 in the counts field, unlike this fixture which has `null`.

Reply:
Fixture was introduced in https://github.com/Crunch-io/crunch-cube/pull/181